smokes/master.cfg

```diff
@@ -61,15 +61,14 @@ c['schedulers'].append(schedulers.ForceScheduler(
 factory = util.BuildFactory()
 # check out the source
 factory.addStep(steps.Git(repourl='https://github.com/buildbot/pyflakes.git', mode='incremental'))
-# run the tests (note that this will not actually run the tests are they do not work with py3)
-factory.addStep(steps.ShellCommand(command=["echo", "trial", "pyflakes"]))
-
+factory.addStep(steps.ShellCommand(command=["trial", "pyflakes"],
+                                   env={"PYTHONPATH": "."}))
 
 slowfactory = util.BuildFactory()
 #check out the source
 slowfactory.addStep(steps.Git(repourl='https://github.com/buildbot/pyflakes.git', mode='incremental'))
-# run the tests (note that this will not actually run the tests are they do not work with py3)
-slowfactory.addStep(steps.ShellCommand(command=["echo", "trial", "pyflakes"]))
+slowfactory.addStep(steps.ShellCommand(command=["trial", "pyflakes"],
```

Comment:
I am not sure we really want to run tests in these smoke tests.This will just make them slower for no reason.

Actually it fails on py3 (https://nine.buildbot.net/#/builders/26/builds/1121/steps/7/logs/stdio). probably the version of pyflakes that we test is not python3 compatible.